Yoast/clicky
# New Clicky tracking code

## Testing
* [Follow the instructions](https://yoast.com/wordpress/plugins/clicky-2/) to get Clicky up and running, but make sure you clone this repository instead of installing a stable zip version.
* Checkout this branch.
* Setup some of the extra settings to ensure they also get outputted.
* Ensure everything still works.

Replaces https://github.com/Yoast/clicky/pull/11. Please ensure you close that PR after testing and merging this one.
